api/src/main/java/uk/gov/hmcts/bar/api/controllers/payment/PaymentInstructionController.java

```diff
@@ -463,11 +450,23 @@ public MultiMap getPIStats(@RequestParam(name = "status", required = true) Payme
         return result;
     }
 
+    @ApiOperation(value = "Send to payhub", notes = "Send all payment-instructions with TTB status to payhub")
+    @ApiResponses(value = {@ApiResponse(code = 200, message = ""),
+        @ApiResponse(code = 400, message = "Bad request"),
+        @ApiResponse(code = 500, message = "Internal server error")})
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping("/send-to-payhub")
```

Comment:
how about 
/payment-instructions/send-to-payhub

Reply:
ok :+1: